client-registry-jempi/importer/postgres/create-db.js

```diff
@@ -0,0 +1,136 @@
+const { Pool } = require('pg');
+
+const user = process.env.POSTGRES_USER || 'postgres'
+const host = process.env.POSTGRES_SERVICE || 'postgres-1'
+const database = process.env.POSTGRES_DATABASE || 'jempi'
+const password = process.env.POSTGRES_PASSWORD || 'instant101'
+const port = process.env.POSTGRES_PORT || 5432
+const newDb = process.env.NEW_DATABASE_NAME || 'jempi'
+const newUser = process.env.NEW_DATABASE_USER || 'keycloak'
```

Comment:
shouldn't this be jempi user to match the .env.cluster configs

Reply:
It maps to that. check the docker compose file https://github.com/jembi/platform/pull/308/files#diff-3452b9e7934f8ae11c4a9771e08c298a7a0f1eea4c703f0c770e39a374e66067R20-R22

Reply:
I actually mean the default value
```suggestion
const newUser = process.env.NEW_DATABASE_USER || 'jempi'
```